redis/commands.py

```diff
@@ -2034,18 +2034,21 @@ def xpending(self, name, groupname):
         """
         return self.execute_command('XPENDING', name, groupname)
 
-    def xpending_range(self, name, groupname, min, max, count,
-                       consumername=None, idle=None):
+    def xpending_range(self, name, groupname, idle=None,
+                       min=None, max=None, count=None,
+                       consumername=None):
         """
         Returns information about pending messages, in a range.
+
         name: name of the stream.
         groupname: name of the consumer group.
+        idle: available from  version 6.2. filter entries by their
```

Comment:
@AvitalFineRedis I don't see idle-time in the function signature, but I do see it in the documentation.

Reply:
The third param